sympy/physics/mechanics/body.py

```diff
@@ -0,0 +1,165 @@
+                 body_inertia=None):
+
+        self.name = name
+        self.loads = []
+
+        if masscenter is None:
+            _masscenter = Point(name + '_masscenter')
+        else:
+            _masscenter = masscenter
+
+        if mass is None:
+            _mass = Symbol(name + '_mass')
+        else:
+            _mass = mass
+
+        if frame is None:
+            _frame = ReferenceFrame(name + '_frame')
+        else:
+            _frame = frame
+
+        if body_inertia is None and mass is None:
+            ixx, iyy, izz, ixz, ixy, iyz = symbols('ixx iyy izz ixz ixy iyz')
+            _inertia = (inertia(_frame, ixx, iyy, izz, ixz, ixy, iyz),
+                        _masscenter)
+        else:
+            _inertia = (body_inertia, _masscenter)
+
+        _masscenter.set_vel(_frame, 0)
+
+        # If user passes masscenter and mass then a particle is created
+        # otherwise a rigidbody. As a result a body may or may not have inertia.
+        if body_inertia is None and mass is not None:
+            self.frame = _frame
+            self.masscenter = _masscenter
+            Particle.__init__(self, name, _masscenter, _mass)
+        else:
+            RigidBody.__init__(self, name, _masscenter, _frame, _mass, _inertia)
+
+    def apply_force(self, vec, point=None):
+        """
+        Adds the force to the point (masscenter by default) on the body.
+
+        Parameters
+        ----------
+        vec: Vector
+            Defines the force vector. Can be any vector w.r.t any frame or
+            combinations of frame.
+        point: Point, optional
+            Defines the point on which the force must be applied. Default is
+            Body's masscenter.
+
+        Example
+        -------
+        To apply a unit force in x direction of body's frame to body's
+        masscenter.
+
+        >>> from sympy import Symbol
+        >>> from sympy.physics.mechanics import Body
+        >>> body = Body('body')
+        >>> g = Symbol('g')
+        >>> body.apply_force(body.mass * g * body.frame.x)
+
+        To apply force to any other point than masscenter, pass that point
+        as well.
+
+        >>> from sympy import Symbol
+        >>> from sympy.physics.mechanics import Body
+        >>> parent = Body('parent')
+        >>> child = Body('child')
+        >>> g = Symbol('g')
+        >>> frame = parent.frame
+        >>> l = Symbol('l')
+        >>> point = child.masscenter.locatenew('force_point', l * body.frame.y)
+        >>> gravity = child.mass * g
+        >>> body.apply_force(gravity * body.frame.x, point)
+
+        """
+        if not isinstance(point, Point):
+            if point is None:
+                point = self.masscenter  # masscenter
+            else:
+                raise TypeError("A Point must be supplied to apply force to.")
+        if not isinstance(vec, Vector):
+            raise TypeError("A Vector must be supplied to apply force.")
+
+        self.loads.append((point, vec))
```

Comment:
We can leave this as is for now, but it seems like the methods are superficial and don't really add much.